### PR TITLE
Add timeouts options for traefik

### PR DESCRIPTION
**Pull request by seuf**
#### What this PR does / why we need it:
Add Traefik [Responding Timeouts](https://docs.traefik.io/configuration/commons/#responding-timeouts) configuration options in traefik chart.

#### Which issue this PR fixes
No issues related.

#### Checklist
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by dtomcej** [comment]
@seuf Can you modify this PR so that those values are not mandatory? You will have to check if they exist before use.

Thanks!

**Comment by seuf** [comment]
> @seuf Can you modify this PR so that those values are not mandatory? You will have to check if they exist before use.
Theses values are already not mandatory. There is already a check if variables are not empty before using it in configMap. Do you want I remove the defaults values (which are also the default of Traefik)
I've rebased over master to fix conflict.

File: stable/traefik/values.yaml [hunk 1]
```diff
@@ -131,6 +131,10 @@ kvprovider:
     #   insecureSkipVerify: true
 
     ## only relevant for etcd
+timeouts:
```
Comment (dtomcej) [inline stable/traefik/values.yaml:134]: These should be optional, not always defined.

File: stable/traefik/templates/configmap.yaml [hunk 2]
```diff
@@ -353,3 +353,13 @@ data:
         {{- end }}
       {{- end }}
     {{- end }}
+    [respondingTimeouts]
```
Comment (dtomcej) [inline stable/traefik/templates/configmap.yaml:356]: These should be optional settings, for backwards compatibility. You will need to test if they exist, and use them if they do, instead of testing if they are empty.

File: stable/traefik/Chart.yaml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 1.56.1
+version: 1.56.2
```
Comment (dtomcej) [inline stable/traefik/Chart.yaml:3]: Since this is a new feature addition, you need to bump the minor version.

**Review by dtomcej — CHANGES_REQUESTED** [review]
